### DISCUSSION TITLE
Mixing your custom home menu with Cooolgamer's custom home menu for even better experience

### DISCUSSION THREAD

**Discussion by Misteryesman** [AromaKitsune/aromakitsune.github.io#3] [Show and tell]
Cooolgamer has an excellent reimagining of the 3DS home menu with translucent menus, updated quick settings icons, Wii U loading symbol, and it messes with the HUD a little. Utilizing the layeredFS, I have combined the two Romfs folders, when combined this way with your sideways icons sleep menu, and clean-looking battery, this is absolutely the best home menu EVER.

If possible it would be awesome if y'all could combine y'all's work or have an option for combined work that includes the sideways icons, sleep menu, and battery. Either way, I absolutely love your work, and I'm so happy you made this available, have been wanting sideways icons for ages!

**Comment by AromaKitsune** [comment]
Can you send your combined RomFS folder (in zip file) here? I could add it to the Custom Home Menu page.

> Either way, I absolutely love your work, and I'm so happy you made this available, have been wanting sideways icons for ages!

Thanks, I'm glad you like it :)

**Comment by Misteryesman** [comment]
> Can you send your combined RomFS folder (in zip file) here? I could add it to the Custom Home Menu page.

[romfs.zip](https://github.com/user-attachments/files/18119022/romfs.zip)

 I also have custom text in my romfs, you can ignore that if you like, but I think its funny

**Reply by somerandomperson1234587** [reply]
How do I install it?

**Reply by AromaKitsune** [reply]
After extracting the zip file contents, RomFS contents go to:
`SD card → /luma/titles/0004003000009802/romfs` for EUR
`SD card → /luma/titles/0004003000008F02/romfs` for USA
`SD card → /luma/titles/0004003000008202/romfs` for JPN

<img width="1100" height="720" alt="image" src="https://github.com/user-attachments/assets/ae278c98-6105-42c9-9319-393cc6c4a195" />

**Reply by somerandomperson1234587** [reply]
BTW if you dont want the custom text just delete the message folder i tried it

**Reply by somerandomperson1234587** [reply]
This is the same romfs file just without the custom text
[romfs.zip](https://github.com/user-attachments/files/24548037/romfs.zip)

**Comment by AromaKitsune** [comment]
This is a great combination of custom home menu assets, nice!

**Comment by Misteryesman** [comment]
when I get the chance i hope to learn how to combine the main menu files of both themes so i can have the ui of Cooolgamer with your spinning icon and slanted games

**Comment by chersbobers** [comment]
This might be late or something but does anyone have a Europe version of the romfs?

**Reply by AromaKitsune** [reply]
Here, only the `theme_intro` one was needed to change. I did not port the custom text though.

[romfs_e.zip](https://github.com/user-attachments/files/21132661/romfs_e.zip)

**Reply by chersbobers** [reply]
Thanks!

**Comment by Kevinw927** [comment]
Hi is it possible to get Battery Percentage shown with this home menu mixing?

**Reply by somerandomperson1234587** [reply]
Idk I can try

**Reply by Kevinw927** [reply]
Would be great. Thanks
